### ISSUE TITLE
Support loading multiple packages with a wildcard

### ISSUE THREAD

**Issue by tobscure** [franzliedke/studio#54]
When configuring a Composer path repository, you can get it to scan a bunch of subdirectories for composer.json files without having to specify any package names:

```json
    "repositories": [
        {
            "type": "path",
            "url": "packages/*"
        }
    ]
```

Could we support something like this in Studio?

Also, is there any reason why the composer package names need to be stored by Studio (as the keys in studio.json), instead of just reading it out of the composer.json file in a package directory at runtime? If not, the studio.json file could take on a new format:

```json
{
    "paths": [
        "packages/*",
        "dir/to/some/other/package"
    ]
}
```

**Comment by franzliedke** [comment]
Okay, I'm half-way done with this. Just need to build some version management for studio.json files. :)

**Comment by franzliedke** [comment]
This is now published in the just-released [0.11.0 version](https://github.com/franzliedke/studio/releases/tag/0.11.0). Just run `studio load packages/*` to load an entire collection of directories with Studio. :)

**Comment by franzliedke** [comment]
Okay, actually it's `studio load "packages/*".
